wavenet.py

```diff
@@ -108,14 +121,15 @@ def _create_network(self, input_batch):
             # Perform (+) -> ReLU -> 1x1 conv -> ReLU -> 1x1 conv to
             # postprocess the output.
             w1 = tf.Variable(tf.truncated_normal(
-                [1, self.residual_channels, int(self.channels / 2)], stddev=0.3,
+                [1, self.residual_channels, self.channels], stddev=0.3,
```

Comment:
I would suggest making the input and output channels here equal to the new skip_channels param, if you went for what I was saying above.

Back on your thread for "generating good audio samples", one of the ideas I mentioned I'd like to try is setting the channels in these last two 1x1/ReLU layers to a larger number, and I think the skip_channels would cleanly fill that role.